Makefile

```diff
@@ -0,0 +1,9 @@
+# wip
+install:
+	docker compose down --volumes \
+	&& rm -rf wp-plugins \
+	&& mkdir -p ./wp-plugins/woocommerce \
+	&& curl -L https://github.com/woocommerce/woocommerce/releases/download/nightly/woocommerce-trunk-nightly.zip -o ./woocommerce-trunk-nightly.zip \
```

Comment:
Perhaps we should consider getting the URL dynamically here as any changes occurring to the URL might effect the make install script.

Reply:
Do you have any suggestions where I that information from? I choose this because this seems te most stable thing out there.

Reply:
You could use Github's API in combination with jq (JSON processor), here's how you could implement that(this is tested and working):
```
install:
	@# Fetch the latest release details from WooCommerce's GitHub repository
	@RELEASE_DATA=$$(curl -s "https://api.github.com/repos/woocommerce/woocommerce/releases"); \
	# Extract the download URL for the nightly build
	NIGHTLY_URL=$$(echo $$RELEASE_DATA | jq -r '.[] | select(.tag_name | contains("nightly")) | .assets[] | select(.name == "woocommerce-trunk-nightly.zip") | .browser_download_url'); \
	docker compose down --volumes; \
	rm -rf wp-plugins; \
	mkdir -p ./wp-plugins/woocommerce; \
	curl -L $$NIGHTLY_URL -o ./woocommerce-trunk-nightly.zip; \
	unzip ./woocommerce-trunk-nightly.zip -d ./wp-plugins; \
	rm ./woocommerce-trunk-nightly.zip; \
	docker compose up

```

Reply:
The only disadvantage about using this, is Github's request limit to 60 requests per hour when the API all is not authenticated

Reply:
That has some limitations. And `jq` is not always available. I think it is important that this is easy to use automation. Also for marketing people to give a WooCommerce Dropday demo.